test/util/genesis/genesis.go

```diff
@@ -141,6 +143,12 @@ func (g *Genesis) WithAppVersion(appVersion uint64) *Genesis {
 	return g
 }
 
+// WithGasPrice sets the gas price of the genesis.
+func (g *Genesis) WithGasPrice(gasPrice float64) *Genesis {
```

Comment:
New functionality to set the gas price for the gentxs (this was failing when I set a high minimum network price in genesis)